### DISCUSSION TITLE
Welcome to cancancan Discussions!

### DISCUSSION THREAD

**Discussion by coorasse** [CanCanCommunity/cancancan#667] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by walterdavis** [comment]
Howdy. I'm Walter Davis, I've been using Rails since 2007 or so, a big user of CanCan since its first iteration (thanks, @rbates). I've used CanCan and CanCanCan in every Rails app I have ever built.

**Comment by SamarthMayya** [comment]
Is there a way I can extract which controller the AccessDenied error is referring to when that exception is thrown? 
![image](https://user-images.githubusercontent.com/67407158/105330641-8a957600-5bf8-11eb-986d-419c87cfb7a4.png)
In the rescue_from code, I want to get the name of the controller. How do I do that?

**Reply by rocket-turtle** [reply]
I might miss something but I think you can do `self.class` in the `rescue_from` block.

**Comment by pooriajr** [comment]
Just wanted to say thank you to the people making this. I used to dread authorization but this gem actually makes it fun!
